DOCS/man/encode.rst

```diff
@@ -153,3 +153,18 @@ You can encode files from one format/codec to another using this facility.
 ``--no-ocopymetadata``
     Turns off copying of metadata from input files to output files when
     encoding (which is enabled by default).
+
+``--ometadata=<metadata-tag[,metadata-tag,...]>``
+    Specifies metadata to include in the output file.
+    Supported keys vary between output formats. For example, Matroska (MKV) and
+    FLAC allow almost arbitary keys, while support in MP4 and MP3 is more limited.
+    You may use an empty value ala ``key=""`` to clear the value (eg. when copying
+    original metadata with ``--ocopymetadata``.)
+
+    .. admonition:: Example
+
+        "``--ometadata title="Output title",comment="Another tag"``"
```

Comment:
I think the examples thus far tend to include "=" even when there is a second level of "=".